vnext/Microsoft.ReactNative/Views/ModalViewManager.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include "ModalViewManager.h"
+
+#include <JSValueWriter.h>
+#include <UI.Xaml.Controls.h>
+#include <Utils/PropertyUtils.h>
+#include <Utils/ValueUtils.h>
+#include <ViewControl.h>
+
+namespace Microsoft::ReactNative {
+
+class ModalShadowNode : public ShadowNodeBase {
+  using Super = ShadowNodeBase;
+
+ public:
+  ModalShadowNode() = default;
+  void createView(const winrt::Microsoft::ReactNative::JSValueObject &) override;
+
+  void AddView(ShadowNode &child, int64_t /*index*/) override {
+    // Not implemented
+    RedBox("Modal is not implemented on RNW Paper");
```

Comment:
Not sure if this is the preferred method to throw an error on Paper? Since Paper and Fabric share a UI Manager, I had to add a ModalViewManager here

Reply:
Can we just not implement a ViewManager?

Reply:
offline discussion, switched to use unimplementedView for Paper